systemtest/src/main/java/io/strimzi/systemtest/utils/TestKafkaVersion.java

```diff
@@ -15,10 +15,29 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class TestKafkaVersion implements Comparable<TestKafkaVersion> {
 
+    private static TestKafkaVersion instance;
+    private static List<TestKafkaVersion> kafkaVersions;
+
+    public static synchronized TestKafkaVersion getInstance() {
```

Comment:
What's the benefit of using singleton here, compared with just having all static methods?

Reply:
I used singleton here to keep parsed kafka-versions file instead of loading it everytime when method is used.

Reply:
You could use a `static { }` block to do that.

Reply:
(aka class initializer, `<clinit>`)

Reply:
I see, let me rework it :)